tests/unit/requirements.txt

```diff
@@ -4,3 +4,4 @@ boto3
 boto
 
 placebo
+parametrize
```

Comment:
We shouldn't need this any longer if the tests are using @pytest.mark.parametrize now